tunneler/tests/test_tunneler.py

```diff
@@ -149,7 +149,7 @@ def test_get_configured_tunnels(self):
         )
 
         configured_tunnels = self.tunneler.get_configured_tunnels()
-        self.assertEqual(configured_tunnels, tunnel_config.keys())
+        self.assertEqual(set(configured_tunnels), set(tunnel_config.keys()))
```

Comment:
Ordering of dict.keys() is not guaranteed, so now testing for content equality without ordering.